lib/containers/utils.pm

```diff
@@ -155,7 +155,9 @@ sub basic_container_tests {
     die("error: missing container $container_name") unless ($output_containers =~ m/$container_name/);
 
     # Containers' state can be saved to a docker image
-    if (script_run("$runtime container exec $container_name zypper -n in curl", 300)) {
+    my $ret = script_run("$runtime container exec $container_name zypper -n in curl", 420);
```

Comment:
I would not be so humble with timeout increase.
Sometimes it's better to have longer timeout, it has time to return reason of the error.
But you know better, great If 420 is good enough. :+1:

Reply:
You're right, I did `600`.